pyrtl/rtllib/aes.py

```diff
@@ -308,3 +436,30 @@ def build_mem(data):
         7c 72 60 6e 44 4a 58 56 37 39 2b 25 0f 01 13 1d 47 49 5b 55 7f 71 63 6d
         d7 d9 cb c5 ef e1 f3 fd a7 a9 bb b5 9f 91 83 8d
         ''')
+
```

Comment:
Everything after this should be moved into an example usage for the docstring at the beginning.